actsync/actionsync.go

```diff
@@ -37,6 +37,8 @@ type (
 		helper   *Helper
 		cfg      Config
 		quit     chan struct{}
+		mu       sync.Mutex
+		stopped  bool
```

Comment:
i feel this flag is redundant, since `IsReady()` already provides an atomic way to start/stop/check the status